samples/azure-quantum/variational-quantum-eigensolver/VQE-qiskit-hydrogen-ionq-sim.ipynb

```diff
@@ -0,0 +1,520 @@
+       "«      └───────────────────────┘└───────────────────────┘└───┘ ║ └╥┘\n",
+       "«c0: 2/════════════════════════════════════════════════════════╩══╩═\n",
+       "«                                                              0  1 </pre>"
+      ],
+      "text/plain": [
+       "      ┌─────────────────────────┐ ┌──────────────────────┐     »\n",
+       "   0: ┤ Ry(-0.0258926310273104) ├─┤ Rz(3.96292972320719) ├──■──»\n",
+       "      └─┬─────────────────────┬─┘┌┴──────────────────────┤┌─┴─┐»\n",
+       "   1: ──┤ Ry(3.3828016387298) ├──┤ Rz(-4.97687159911975) ├┤ X ├»\n",
+       "        └─────────────────────┘  └───────────────────────┘└───┘»\n",
+       "c0: 2/═════════════════════════════════════════════════════════»\n",
+       "                                                               »\n",
+       "«      ┌───────────────────────┐┌───────────────────────┐     »\n",
+       "«   0: ┤ Ry(-2.92184532966861) ├┤ Rz(-2.59110067976668) ├──■──»\n",
+       "«      └┬──────────────────────┤├───────────────────────┤┌─┴─┐»\n",
+       "«   1: ─┤ Ry(1.01813740573041) ├┤ Rz(-2.37222153727425) ├┤ X ├»\n",
+       "«       └──────────────────────┘└───────────────────────┘└───┘»\n",
+       "«c0: 2/═══════════════════════════════════════════════════════»\n",
+       "«                                                             »\n",
+       "«       ┌──────────────────────┐┌───────────────────────┐     »\n",
+       "«   0: ─┤ Ry(3.35072313484003) ├┤ Rz(-3.57591366441145) ├──■──»\n",
+       "«      ┌┴──────────────────────┤├───────────────────────┤┌─┴─┐»\n",
+       "«   1: ┤ Ry(-2.33565366149536) ├┤ Rz(-4.02171876301366) ├┤ X ├»\n",
+       "«      └───────────────────────┘└───────────────────────┘└───┘»\n",
+       "«c0: 2/═══════════════════════════════════════════════════════»\n",
+       "«                                                             »\n",
+       "«       ┌──────────────────────┐ ┌──────────────────────┐┌───┐┌─┐   \n",
+       "«   0: ─┤ Ry(3.30633207147813) ├─┤ Rz(3.75248329183843) ├┤ H ├┤M├───\n",
+       "«      ┌┴──────────────────────┤┌┴──────────────────────┤├───┤└╥┘┌─┐\n",
+       "«   1: ┤ Ry(-1.50980520494039) ├┤ Rz(-2.08578439227638) ├┤ H ├─╫─┤M├\n",
+       "«      └───────────────────────┘└───────────────────────┘└───┘ ║ └╥┘\n",
+       "«c0: 2/════════════════════════════════════════════════════════╩══╩═\n",
+       "«                                                              0  1 "
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "circ.assign_parameters(ret.optimal_parameters).draw()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4d49d174",
+   "metadata": {},
+   "source": [
+    "## 2. Run on IonQ simulator via Azure Quantum\n",
+    "\n",
+    "Now, we can connect to Azure quantum and run VQE on the hardware backends."
```

Comment:
```suggestion
    "Now, we can connect to the Azure Quantum service and run VQE on the hardware backends."
```

Reply:
I believe we call our product Azure Quantum.

Reply:
Yes, the whole product, including the service, the Quantum Development Kit, and several other components.

Reply:
I just checked and according to PM:
> Our product is called "Azure Quantum"

Reply:
I suggest using "Azure Quantum Workspace" here to be syntactically correct.

Reply:
I don't disagree that our product is called "Azure Quantum," but this sentence refers to a part of our product. For clarity and for consistency with our documentation, my comment remains as above.